frontend/components/FileUpload/FileUpload.css

```diff
@@ -16,4 +16,22 @@
 .dropzone {
   align-items: center;
   display: flex;
+  padding: 10px;
+  border: 1px solid transparent;
+  border-radius: 2px;
+}
+
+.dropzone-active {
+    border: 1px solid var(--theme-colour-data, #4A91FF);
+    background-color: #e3eeff;
+    box-shadow: 0 0 14px rgba(0,0,0,.1) inset;
+    -webkit-transition: box-shadow 300ms linear;
```

Comment:
@eduardoboucas ah, indeed we do. There was no override, but that's a cleaner declaration